metaflow/plugins/kfp/kfp.py

```diff
@@ -564,6 +565,34 @@ def _step_cli(self, node: DAGNode, task_id: str, user_code_retries: int) -> str:
         cmds.append(" ".join(entrypoint + top_level + step))
         return " && ".join(cmds)
 
+    @staticmethod
+    def _create_resource_based_node_type_toleration(
```

Comment:
can you please add a comment like
`# TODO(yunw): A Metaflow plugin framework to customize tags, labels, etc.`

Reply:
also reference `AIP-5264 MutatingWebHook (or Validating) for AIP pod scheduling policies`

Reply:
@talebzeghmi we had PR for label and annotation but decided to close it to not leave interface for customer: https://github.com/zillow/metaflow/pull/59. Do you think we should add back these decorator?

Reply:
Same as the other comment below. Will add in separate PR.